lib/helpers/reports/convertStandups.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const _                   = require('underscore');
+const standupReport       = require('../getStandupReport');
+
+module.exports = function createAttachments(standups) {
+  let attachments = [];
+
+  _.each(standups, function (standup) {
+    attachments.push(standupReport(standup));
+  });
```

Comment:
This comment reveals that I'm trying to put more into this PR than it's supposed to have.  I apologize for that!  You've done a lot of good work in here and I don't want to derail that because there's still work to do.